client/ayon_traypublisher/plugins/create/create_texture.py

```diff
@@ -0,0 +1,281 @@
+            if strip_common_suffix:
+                # Remove common suffix from the variant name
+                basename = basename.removesuffix(common_suffix)
+
+            if variant_name_method == "filename":
+                # Use the filename as the variant name
+                variant = basename
+            elif variant_name_method == "variant":
+                variant = instance_data["variant"]
+            elif variant_name_method == "filename_rsplit_underscore":
+                # Use the filename after the last underscore as the
+                # variant name. (Note that we stripped any underscore before
+                # the UDIM tile/sequence number already).
+                variant = basename.rsplit("_", 1)[-1]
+            else:
+                raise ValueError(
+                    f"Unknown variant name method: {variant_name_method}"
+                )
+
+            # Apply variant prefix/suffix if specified
+            variant = "{}{}{}".format(prefix, variant, suffix)
+
+            # Create instance
+            product_name = self.get_product_name(
+                self.project_name,
+                folder_entity,
+                task_entity,
+                variant,
+                host_name=self.host_name,
+            )
+            _pre_create_data = copy.deepcopy(pre_create_data)
+            _pre_create_data["representation_files"] = file_info
+            _instance_data = copy.deepcopy(instance_data)
+            _instance_data["variant"] = variant
+            self._create_instance(
+                product_name=product_name,
+                instance_data=_instance_data,
+                pre_create_data=_pre_create_data
+            )
+
+    def _create_instance(self, product_name, instance_data, pre_create_data):
+        repr_file = pre_create_data.get("representation_files")
+        if not repr_file:
+            raise CreatorError("No files specified")
+
+        # Pass pre-create attributes to instance creator attributes
+        instance_data["creator_attributes"] = pre_create_data
+
+        # Trigger the dedicated `texture_creator` collector plug-in
+        instance_data.setdefault("families", []).append("texture_creator")
+
+        # Create new instance
+        new_instance = CreatedInstance(self.product_type, product_name,
+                                       instance_data, self)
+        self._store_new_instance(new_instance)
+
+    def _get_udim_attr_def(self) -> BoolDef:
+        return BoolDef(
+            "is_udim",
+            default=True,
+            label="UDIM",
+            tooltip=(
+                "Define whether the input files are a UDIM sequence "
+                "instead of frame range sequence.\n"
+                "This is only relevant if the input files is a sequence "
+                "of files."
+            )
+        )
+
+    def _get_review_attr_def(self) -> BoolDef:
+        return BoolDef(
+            "add_review_family",
```

Comment:
This is just suggestion
```suggestion
            "mark_for_review",
```

Reply:
This is just copied from the others, like:
```
ayon_traypublisher/plugins/create/create_texture.py
ayon_traypublisher/plugins/create/create_online.py
ayon_traypublisher/plugins/create/create_movie_batch.py
ayon_traypublisher/plugins/create/create_editorial_simple.py
ayon_traypublisher/plugins/create/create_editorial_advanced.py
ayon_traypublisher/plugins/create/create_csv_ingest.py
```